stories/ExampleSlider.jsx

```diff
@@ -1,8 +1,8 @@
 import React, { PropTypes } from 'react';
 import { storiesOf } from '@kadira/storybook';
 
-import Rheostat from '../';
-import log10 from '../lib/algorithms/log10';
+import Rheostat from '../src/Slider'; // TODO PW: Change back to ../
```

Comment:
Honestly, I think it makes more sense for the storybook to use the src files instead of the built files. :)